### PR TITLE
Tablet background colour

### PR DESCRIPTION
**Pull request by nowarkdev**
Fixes tablet background color so that main is white and faqs and bellow is athens-grey

### CODE REVIEW

File: src/components/layout.scss [hunk 1]
```diff
@@ -39,6 +42,7 @@ $faqsWidthDesktop: 25%;
     padding: 15px;
     box-sizing: border-box;
     float: left;
+    background-color: white;
```
Comment (johnsibly) [inline src/components/layout.scss]: this should be a variable @frikille

File: src/components/layout.scss [hunk 2]
```diff
@@ -11,6 +11,9 @@ $faqsWidthDesktop: 25%;
   position: relative;
   min-height: $heightOfPageContent;
   background-color: $athens-grey-zeus;
+  @include tablet {
+    background-color: $athens-grey;
```
Comment (johnsibly) [inline src/components/layout.scss]: Why is it athens-grey on tablet/desktop but athens-grey-zeus on mobile? @benstokoe

File: src/components/layout.scss (hunk 1 above)
Comment (nowarkdev) [inline src/components/layout.scss]: Why would you have variables for white or black, surely they are always white and black? @bdo @rogerfranklin2

File: src/components/layout.scss (hunk 2 above)
Comment (nowarkdev) [inline src/components/layout.scss]: We're not sure but this is what Vic has asked us to do...
@rogerfranklin2 @bdo

File: src/components/layout.scss (hunk 1 above)
Comment (peter-mouland) [inline src/components/layout.scss]: i'd agree, just for consistency. I think if we have it as a rule `style-guide should always use the colours variables` this is easy to follow.  It less likely that people start using their own 'tweaked' colours in the future.

Comment (bdo) [inline src/components/layout.scss]: I think that rule still applies. We're talking about a very special color here guys: *white*

Comment (peter-mouland) [inline src/components/layout.scss]: there is already a `$white` and `$black` within `utils/_colours.scss`.  As this is a project with *lots* of contributors having simple rules to follow makes things smoother and easier in the long run

Comment (bdo) [inline src/components/layout.scss]: fair enough, we'll use that - we might close and redo this pull request anyway ;)

**Comment by nowarkdev** [comment]
Closing as we will recreate this fix without a breaking change

**Pull request closed without merging**